src/gp/src/GPMSAOptions.C

```diff
@@ -554,9 +554,12 @@ GPMSAOptions::set_final_scaling
         min_max_update(minScenario, maxScenario,
                        (*m_simulationScenarios[i]));
 
+      // To match Matlab code, only use simulation data for scaling.
```

Comment:
I failed to come up with a decent middle ground there, huh?  I was trying to avoid both the confusing Scylla of "just delete the code, and leave future generations with no indication of why we're ignoring half of our input data" and the passive-aggressive Charybdis of "// Only use simulation data for scaling.  This is necessary to match Matlab code output, not for any sane reason."  ;-)

Would it be reasonable for me to leave the code in, but wrapped in an "autoscale_including_experiment_data" option (which defaults to false, if you like)?  Or should I move the documentation to the autoscale method declarations?  Or both?  (I'm now leaning toward "both".)

I'm seriously insisting on *some* sort of documentation to exist regarding why we're making this choice.  "Why are we normalizing numbers to a (min,max) range which will leave numbers outside that range?" is a question with a defensible answer, but "Why are we doing so without telling users?" is not.